pkg/ccl/changefeedccl/schemafeed/schema_feed.go

```diff
@@ -592,31 +592,46 @@ func (tf *schemaFeed) fetchDescriptorVersions(
 		log.Infof(ctx, `fetching table descs (%s,%s]`, startTS, endTS)
 	}
 	start := timeutil.Now()
+
+	// TODO: This should probably process these as they're fetched instead and be
+	// memory monitored, etc.
```

Comment:
// TODO: Loop should probably process partial SSTs as they're fetched instead and be
	// memory monitored, etc.

Is this a fair interpretation? Unless it is not meant to stay.

Reply:
I don't know if they're really "partial SSTs" per se, just batches of descriptor revisions, that happen to be in SST format as a means of transport. This TODO is just here since while I batched the fetching, I didn't go so far as to batch the processing.